src/components/views/dialogs/ConfirmKeyStorageOffDialog.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+import React from "react";
+import ErrorIcon from "@vector-im/compound-design-tokens/assets/web/icons/error";
+import { Button } from "@vector-im/compound-web";
+import { PopOutIcon } from "@vector-im/compound-design-tokens/assets/web/icons";
+
+import { _t } from "../../../languageHandler";
+import defaultDispatcher from "../../../dispatcher/dispatcher";
+import { EncryptionCard } from "../settings/encryption/EncryptionCard";
+import { EncryptionCardButtons } from "../settings/encryption/EncryptionCardButtons";
+import { type OpenToTabPayload } from "../../../dispatcher/payloads/OpenToTabPayload";
+import { Action } from "../../../dispatcher/actions";
+import { UserTab } from "./UserTab";
+
+interface Props {
+    onFinished: (dismissed: boolean) => void;
+}
+
+export default class ConfirmKeyStorageOffDialog extends React.Component<Props> {
```

Comment:
doc-comment, please. What does this do, and how is it used?

Reply:
Fixed in c7decffde7